### DISCUSSION TITLE
소스 다운 받았습니다

### DISCUSSION THREAD

**Discussion by niceguykim** [yoonjeong-kwon/fastcampus-kubernetes#9] [Q&A] [answered]
git clone시에 파일 체크아웃 실패로 파일이 다운로드가 되지 않습니다 
파일명 앞에 특수문자 들어 있는 것 같은데 파일명 바꿔주세요

**Comment by yoonjeong-kwon** [comment, marked as answer]
@niceguykim 님 안녕하세요. 메일 분류가 잘못되어 메일이 온 것을 빠르게 확인하지 못했네요. 
늦어져서 죄송합니다.
git clone 시 발생하는 에러 메시지를 남겨주실 수 있을까요? 
파일명 앞에 특수문자가 들어가 있는 파일은 없어서요. 현재 git clone도 정상적으로 동작하고 있습니다.
메시지 남겨주시면 확인 후 답변 드리겠습니다.

**Comment by niceguykim** [comment]
안녕하세요 강사님
에러는 지웠는데 검색해 보니 이미지파일 이름 앞에 특문 있면 윈도우에서는 clone안된다고 하더라고요

압축 파일 내려 받아서 문제는 없으니 신경 안쓰셔도 됩니다
등록글 삭제 방법을 몰라서 놔뒀는데 무시하셔도 됩니다~^^

2022년 7월 17일 (일) 오후 10:48, yoonjeong-kwon ***@***.***>님이 작성:

**Reply by yoonjeong-kwon** [reply]
네 답변 감사합니다^_^
